test/test.js

```diff
@@ -639,6 +640,23 @@ QUnit.test('AltField and AltFormat tests', function( assert ) {
     
     assert.equal($(SecondaryAltField).val(), '', "The secondary field was cleared.");
     
+    field.MonthPicker('option', 'SelectedMonth', '06/2016');
+    
+    assert.equal(field.val(), '06/2016', 'The main field was populated correctly using the SelectedMonth option.');
+    assert.equal($(SecondaryAltField).val(), '06/2016', "The secondary field was populated correctly using the SelectedMonth option.");
+    
+    field.MonthPicker('option', 'SelectedMonth', null);
+    
+    assert.equal(field.val(), '', 'The main field was cleared by passing null to the SelectedMonth option.');
+    assert.equal($(SecondaryAltField).val(), '', "The secondary field was cleared by passing null to the SelectedMonth option..");
+
+    assert.equal(field.MonthPicker('GetSelectedMonthYear'), null, 'GetSelectedMonthYear API call returned null when there was no date selected as expected.');
```

Comment:
I remember looking at this code in the morning on the way to work and asking myself "where is date validation that makes the validating message visible being performed?".

After some debugging I was once again surprises to be reminded that the `GetSelectedMonthYear` method in addition to returning a string representing the selected month/year it has visual side effects that triggers date validation.

The person reading this code has no way to know that in addition to checking the value returns from the `GetSelectedMonthYear` method this line is required to make the validation message become visible.

After stumbling upon this method for the second time ([scroll to the end of this comment to see the first time](https://github.com/KidSysco/jquery-ui-month-picker/pull/17#issue-109564591)) I don't understand way I didn't at least suggest to rename and deprecate this method in version 3.

Perhaps we should:
* Use the `Validate()` method (which does exactly the same thing except it returns a JavaScript Date object.
* Add a comment explaining that the purpose of calling this method is to trigger visual validating.
* Test the value returned from the `Validate()` method in a separate line.